README.md

```diff
@@ -1,22 +1,52 @@
-# UF OSC Website
+# UF Open Source Club Website
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme) ![GitHub issues](https://img.shields.io/github/issues-raw/ufosc/Club_Website_2) ![GitHub](https://img.shields.io/github/license/ufosc/Club_Website_2) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com) ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/ufosc/Club_Website_2/Node.js%20CI) ![GitHub package.json version](https://img.shields.io/github/package-json/v/ufosc/Club_Website_2)
 <br/>
-Website for the UF Open Source Club.
+The official website of the UF Open Source Club. 
 
 ## Table of Contents
+- [Background](#background)
 - [Install](#install)
 - [Usage](#usage)
 - [Configuring](#configuring)
+- [Maintainers](#maintainers)
 - [Contributing](#contributing)
 - [License](#license)
 
+## Background
+Started in the Fall semester 2022, this website is built and maintained by club members to serve as a hub of information for the club.
```

Comment:
Too short to be in its own separate section, please move it between the title & table of contents section.